.github/workflows/nodejs.yml

```diff
@@ -19,6 +19,10 @@ jobs:
 
       - name: Install deps and build (with cache)
         uses: bahmutov/npm-install@v1
+      - name: Install deps and build (with cache) - website
+        uses: bahmutov/npm-install@v1
+        with:
+          working-directory: website
```

Comment:
This seems very unrelated and the new website Jared added seems to have its own system set-up from CI (puts comments literally everywhere as seen in this PR, which is mildly annoying especially when no changes to the website were made...). I saw you wrote this:

> Also fixed _Node CI / Lint on node 10.x and ubuntu-latest (pull_request)_ check.

However if this fixes it, that doesn't seem to be intentional. There's also a change required to `src/index.ts` to fix the lint issue, so this seems to break things.
The correct fix is to just run `yarn lint --fix` -- I did so in a separate PR: #794

Reply:
Merged #794 now which fixes the lint errors properly. So if you remove this and rebase, Lint should still pass
```suggestion
```

Reply:
Removed it myself